src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
-    @GetMapping("/greet")
-    public String greet(@RequestParam(value = "name", defaultValue = "World") String name) {
```

Comment:
2. The method `greeting` accepts a request parameter `name` with a default value of "World" if not provided. It returns a greeting message using the provided name.